src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -459,7 +461,9 @@ private function getOrderProducts(Order $order): OrderProductsForViewing
                 $this->locale->formatPrice($product['amount_refund'], $currency->iso_code),
                 $product['product_quantity_refunded'],
                 $this->locale->formatPrice($product['displayed_max_refundable'], $currency->iso_code),
-                $product['location']
+                $product['location'],
+                !empty($product['id_order_invoice']) ? $product['id_order_invoice'] : null,
```

Comment:
Here, you check the existence of `id_order_invoice ` but not when you `$orderInvoice` or display the number, what will happen for a product without invoice then?

Reply:
The value goes in a json, which is passed throught methods in javascript, but is checked before being used, in order-product-edit.js.

Reply:
I meant the last field, `$orderInvoice->getInvoiceNumberFormatted($order->id_lang)` If `id_order_invoice ` is empty then the `$orderInvoice` is initialized with an id `0` so the returned data doesn't make sense The last field should be null as well then no?

Reply:
Ah yes, I thought you were talking about the field above 👍

Reply:
(done)